syntax = "proto3";

package ecclesia;

message HttpCredential {
  string username = 1;
  string password = 2;
}
